src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
I think the JsObject part was the most confusing angle to me. I think I get it now. Why aren't WorkflowOptions just simple string key/value pairs? Is it to allow for a world where more complicated structures are being taken in? If so, can you envision a use case for that?